properties/P000203.md

```diff
@@ -0,0 +1,16 @@
+---
+uid: P000203
+name: Almost discrete
+refs:
+- zb: "1542.54005"
+  name: On the product of almost discrete Grothendieck spaces (Osipov)
+---
+
+All points except for one are isolated.
+
+Equivalently, the topology is not {P52} and is finer than an excluded point topology (e.g. {S12}).
```

Comment:
```suggestion
Equivalently, the topology is not {P52} and contains an excluded point topology (e.g. {S12}).
```
so that we don't need to say "finer or equal"